packages/round-manager/src/features/round/ViewRoundPage.tsx

```diff
@@ -63,6 +63,11 @@ export default function ViewRoundPage() {
       setRoundExists(!!round);
 
       if (round) {
+        /* During development, give frontend access to all rounds */
+        if (process.env.NODE_ENV === "development") {
```

Comment:
We might want to test that round access works in development, so using a debug mode environment flag or a similar alternative instead of the Node env is a better approach here. What do you think?

Reply:
I dont' see your point. You can set the NODE_ENV variable yourself if you want to change it during testing or on CI.

Reply:
Yes, you can change it. But my point is that they mean completely different things, the `NODE_ENV` specifies the environment you're currently running on and you're trying to disable some feature to help test. These two things are not related, because there will be situations where you might want to test this specific feature you're disabling while keeping the `NODE_ENV` as `development`. You're also breaking feature parity with production for the whole development team, this will be confusing for team members that don't know this exists, it is safer to have the same features as production by default and disable them manually by changing your env.

Reply:
Ah now I get it - fixed by introducing a new IGNORE_FRONTEND_CHECKS environment variable.